x/dex/types/keys.go

```diff
@@ -62,14 +64,26 @@ func TokenMapKey(address string) []byte {
 	return key
 }
 
+func TickIndexToBytes(tickIndex int64) []byte {
+	key := make([]byte, 9)
```

Comment:
You can use `[9]byte{}` if you know ahead of time the length of the byte array.

Reply:
This doesn't work. We continue to append onto pairId in `TickKey` so it has to be a dynamic length